packages/phisherman/src/lib/Phisherman.ts

```diff
@@ -56,6 +56,52 @@ export function reportDomain(domain: string, apiKey: string = storedApiKey) {
 	);
 }
 
+/**
+ * Returns information for a domain.
+ * @param domain The domain to get info about.
+ * @param apiKey optionally pass a Phiserman API key for making this request. This will default to {@link storedApiKey}, which can be configured through {@link setApiKey}.
+ * @since 1.1.0
+ */
+export async function getDomainInfo(domain: string, apiKey: string = storedApiKey) {
+	const result = await fetch<PhishermanInfoType>(
+		`https://api.phisherman.gg/v2/domains/info/${domain}`,
+		{
+			headers: {
+				'Content-Type': 'application/json',
+				'User-Agent': `Sapphire Phisherman/1.0.0 (node-fetch) ${os.platform()}/${os.release()} (https://github.com/sapphiredev/utilities/tree/main/packages/phisherman)`,
```

Comment:
Can you extract the user agent to a not exported constant? Now it's really being repeated a lot.